src/main/java/com/tsunderebug/speedrun4j/game/run/Player.java

```diff
@@ -1,31 +1,32 @@
 package com.tsunderebug.speedrun4j.game.run;
 
+import com.tsunderebug.speedrun4j.user.NameStyle;
 import com.tsunderebug.speedrun4j.user.User;
 
-import java.awt.*;
-import java.io.IOException;
-
 public class Player {
 
 	private String rel;
 	private String id;
 	private String name;
 	private String uri;
 
-	public String getName() throws IOException {
-		if(id != null) {
-			return User.fromID(id).getNames().get("international");
-		} else {
-			return name;
-		}
+	public String getName() {
+		return id != null ? User.fromID(id).getNames().get("international") : name;
```

Comment:
Just a nitpick, I would personally prefer ternaries to only be used when both sides are identifiers / pure values.